server/app/controllers/admin/AdminReportingController.java

```diff
@@ -57,31 +57,40 @@ private CiviFormProfile getCiviFormProfile(Http.Request request) {
 
   @Secure(authorizers = Authorizers.Labels.ANY_ADMIN)
   public Result show(Http.Request request, String programSlug) {
-    String programName =
-        programService
-            .getActiveFullProgramDefinitionAsync(programSlug)
-            .toCompletableFuture()
+    String programLocalizedName =
+        programRepository
+            .getActiveProgramFromSlug(programSlug)
             .join()
-            .adminName();
-
+            .getProgramDefinition()
+            .localizedName()
+            .getDefault();
+    String programAdminName =
```

Comment:
**question:** It doesn't look like there's any difference with the `programLocalizedName` and `programAdminName` with how they're currently written?

Reply:
They are different. I have added screenshots in the description for clarity.